client/app/components/queries/SchedulePhrase.jsx

```diff
@@ -51,7 +51,7 @@ export default class SchedulePhrase extends React.Component {
     const content = full ? <Tooltip title={full}>{short}</Tooltip> : short;
 
     return this.props.isLink ? (
-      <a className="schedule-phrase" onClick={this.props.onClick} data-test="EditSchedule">
+      <a className="schedule-phrase" role="link" tabIndex={0} onClick={this.props.onClick} data-test="EditSchedule">
```

Comment:
`a` should already have the role `link` -- I don't believe you need to add this attribute.

Reply:
Followup: without the `href` attribute, this has no role. With the an `href`, it is a `link`. 

Source: https://w3c.github.io/html-aria/#el-a